### ISSUE TITLE
-q (--sequential) option seems to be not working

### ISSUE THREAD

**Issue by annahake** [stephaneguindon/phyml#104]
Hi, 
I use phyml Version 3.3.20190321 (from bioconda) from the command line. 
When I start phyml with a sequential alignment file like your example [file](http://www.atgc-montpellier.fr/phyml/primatesNT.phy) and with the option -q or --sequential, the program starts with 
` Command line: phyml -i primatesNT.phy -m GTR --sequential 
`
end never finishes or actually starts running or giving any error code. 
If I change the format to interleaved everything works.

**Comment by stephaneguindon** [comment]
Indeed... The github version works fine with the --sequential and -q options but the bioconda one seems to fail miserably... Maybe @epruesse or @annahake have an idea?

**Comment by epruesse** [comment]
No clue... what does the option do?

@annahake You could try running `gdb --args phyml....` and then `run` in the prompt `gdb` opens and then after a while press Ctrl-C to get back into the debugger and then see with `bt` command where in the code it is. Repeat for sampling.

**Comment by stephaneguindon** [comment]
--sequential aka -q set the value of io->interleaved to NO (the default being YES).  PhyML then calls Read_Seq_Interleaved(io) (in case io->interleaved == YES) or Read_Seq_Sequential(io) (when io->interleaved == NO). Those two functions have been here forever and I don't imagine there could be any serious issue with them. I'm perplexed...

**Comment by epruesse** [comment]
Can you reproduce it?

**Comment by epruesse** [comment]
And, is this osx or linux or both?

**Comment by stephaneguindon** [comment]
I was able to reproduce the bug on ubuntu 18.04. I haven't tried any other architecture.

**Comment by annahake** [comment]
I tested it also on a linux machine.
